reference/library/src/morpheus-master/sass/base/_bootstrap-defaults.scss

```diff
@@ -145,6 +145,25 @@ $icon-font-path: if($bootstrap-sass-asset-helper, "bootstrap/", "../" + $skin-na
 //** Border color for table and cell borders.
 // $table-border-color:            #ddd
 
+// SAK-32144 override the nowrap behavior of table-responsive
+.table-responsive {
+  @media screen and (max-width: 480px) {
```

Comment:
Do we want to use the bootstrap variables in out definitions? So this would use `$screen-xs`, but it's deprecated.

So looking a little more should we be using the bootstrap grid sizes for most of these limit and so say anything less that `768px`is xtra small mobile and so have the media query be `(max-width: $screen-xs-max)`

https://getbootstrap.com/css/#grid